db/custom_seeds/notifications.rb

```diff
@@ -0,0 +1,397 @@
+# Notification seed file, to mock notifications for a specific user (ideally yourself).
+# To run the command, run:
+# rails db:seed:notifications USERNAME='your_username'
+# you can also add CLEAN_SETUP="y" if you want to start with fresh notifications.
+
+require "#{Rails.root}/db/seeder"
+
+NUM_TOTAL_STEPS = 11
+
+user = User.find_by(username: ENV["USERNAME"])
+if user.blank?
+  puts "Your user not found. Please ensure the username is present or that the command was properly run: rails db:seed:notifications USERNAME='your_username'"
+  raise StandardError
+end
+
+if ENV["CLEAN_SETUP"].present?
+  puts "starting with none of your existing notifications, followers, articles, and comments..."
+  user.notifications.delete_all
+  Follow.where(followable: user).destroy_all
+  Follow.where(follower: user).destroy_all
+  user.articles.destroy_all
+  user.comments.destroy_all
+end
+
+seeder = Seeder.new
+
+puts "0/#{NUM_TOTAL_STEPS} checking requirements..."
+users = User.where(id: 1..6).count == 6 ? User.where(id: 1..6) : []
+((1..6).to_a - users.pluck(:id)).each do |id|
+  user = User.create!(
+    id: id,
+    name: name,
```

Comment:
`name` here needs to be set, it is not defined. Probably want this at the top of the block the `create!`
```ruby
name = Faker::Name.unique.name
```